### DISCUSSION TITLE
红米ax6路由器下哪个包呀

### DISCUSSION THREAD

**Discussion by a13720943940** [messense/aliyundrive-webdav#169] [Q&A] [answered]
红米ax6路由器下哪个包呀

**Comment by messense** [comment, marked as answer]
opkg print-architecture 看下输出。

**Reply by a13720943940** [reply]
输出这样的
root@OpenWrt:~# opkg print-architecture
arch all 1
arch noarch 1
arch aarch64_cortex-a53_neon-vfpv4 10


我下载了这3个都不行
aliyundrive-webdav_1.0.0_aarch64_cortex-a53.ipk
aliyundrive-webdav_1.0.0_aarch64_cortex-a72.ipk
aliyundrive-webdav_1.0.0_aarch64_generic.ipk

**Reply by a13720943940** [reply]
> opkg print-architecture 看下输出。

在线等，麻烦了，谢谢

**Reply by a13720943940** [reply]
> opkg print-architecture 看下输出。

输出这样的
root@OpenWrt:~# opkg print-architecture
arch all 1
arch noarch 1
arch aarch64_cortex-a53_neon-vfpv4 10

我下载了这3个都不行
aliyundrive-webdav_1.0.0_aarch64_cortex-a53.ipk
aliyundrive-webdav_1.0.0_aarch64_cortex-a72.ipk
aliyundrive-webdav_1.0.0_aarch64_generic.ipk

**Reply by messense** [reply]
`/etc/opkg.conf` 中增加 

```
arch all 1
arch noarch 1
arch aarch64_cortex-a53_neon-vfpv4 10
arch aarch64_cortex-a53 100
```

保存后安装 aliyundrive-webdav_1.0.0_aarch64_cortex-a53.ipk

**Reply by a13720943940** [reply]
> `/etc/opkg.conf` 中增加 
> 
> ```
> arch all 1
> arch noarch 1
> arch aarch64_cortex-a53_neon-vfpv4 10
> arch aarch64_cortex-a53 100
> ```
> 
> 保存后安装 aliyundrive-webdav_1.0.0_aarch64_cortex-a53.ipk

感谢大佬，感激不尽，我这就试试
